test/curl_converter_test.dart

```diff
@@ -27,6 +27,97 @@ void main() {
     );
   }, timeout: defaultTimeout);
 
```

Comment:
Add more tests for Curl.parse 
which handle fields with array values, headers, etc.